packages/api/src/router/deployment-variable.ts

```diff
@@ -149,114 +143,127 @@ const valueRouter = createTRPCRouter({
         })
         .returning();
     }),
+});
+
+export const deploymentVariableRouter = createTRPCRouter({
+  value: valueRouter,
 
   byTargetId: protectedProcedure
+    .meta({
+      authorizationCheck: ({ canUser, input }) =>
+        canUser
+          .perform(Permission.DeploymentGet)
+          .on({ type: "target", id: input }),
+    })
     .input(z.string().uuid())
     .query(async ({ ctx, input }) => {
-      const variableValuesQuery = ctx.db
```

Comment:
we had this variable.values.byTargetId that I removed
basically seems like we wanted to get the variables within a deployment that a target had values for + any possible value that the target could be assigned to for that variable, which I ended up juist doing in the `variable.byTargetId` endpoint